message-me/app/views/chatrooms/index.html.erb

```diff
@@ -1 +1,62 @@
-<h1>this is index.html.erb for chatroom</h1>
+<h4 class="ui center aligned medium icon header">
+  <i class="orange circular coffee icon"></i>
+  Say Something
```

Comment:
Don't leave this like this. It makes it tough on the structure.
If the child of an element has a html node, then make sure all has a node.